web-client/src/presenter/presenter.ts

```diff
@@ -1444,6 +1441,7 @@ export const presenterSequences = {
     updateDocketNumberSearchFormSequence as unknown as Function,
   updateFileDocumentWizardFormValueSequence:
     updateFileDocumentWizardFormValueSequence as unknown as Function,
+  updateFilePetitionSequence: updateFilePetitionSequence as unknown as Function,
```

Comment:
as unknown as Function is for legacy sequences. Would you mind updating the type so that we know if this sequence needs any props? If you need examples take a look at 

docs/typesOnTSX.md